src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
gdb at least often needs to be invoked with some arguments.
These are args passed directly to gdb, not to the client application we are launching to debug.